charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 56.2.1
+version: 56.2.2
```

Technically, its a new feature, not a bug fix. Please increment the minor version

```suggestion
version: 56.3.0
```